iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/Substations.java

```diff
@@ -27,7 +27,7 @@ private Substations() {
             return substations;
         }
         return Iterables.filter(substations, substation -> {
-            if (country != null && country != substation.getCountry()) {
+            if (country != null && country != substation.getCountry().orElse(null)) {
```

Comment:
How do I get substations without country. How should be considered optional countries?